components/efuse/esp32s3/esp_efuse_rtc_calib.c

```diff
@@ -24,7 +36,7 @@ int esp_efuse_rtc_calib_get_ver(void)
 
     uint32_t cali_version_v1 = (blk_ver_major == 1) ? 1 : 0;
     if (!cali_version_v1) {
-        ESP_LOGW("eFuse", "calibration efuse version does not match, set default version: %d", 0);
+         ESP_LOGW("eFuse", "calibration efuse version does not match, set default version: %d", 0);
```

Comment:
extra space ;)

Reply:
Oops, sorry about that.  I have fixed this.

Reply:
@LucasTambor PTAL when you have a chance.